app/controllers/wikis_controller.rb

```diff
@@ -1,5 +1,9 @@
 class WikisController < ApplicationController
   def index
+      if ! current_user.present? || current_user.standard?
```

Comment:
@DarylMcCullough We want to limit this with our Policy scoping. See more below in WikiPolicy! 

`@wikis = policy_scope(Wiki)`